flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -168,6 +171,7 @@ class AndroidNotificationDetails {
       this.largeIcon,
       this.largeIconBitmapSource,
       this.onlyAlertOnce,
+      this.showWhen,
```

Comment:
Given the default behaviour is to show the timestamp, making the default value true will make this more obvious to users of the plugin